playground/cli/simple-pdf-to-json/newjson2pdf.js

```diff
@@ -0,0 +1,70 @@
+//'{"name": "Naval","twitterHandle": "naval", "city":"New York","image":"abs.jpg","tweet": "Tweet1","createAt": "2018-05-31"}')
+
+//parse json before using it as data
+var data = [
```

Comment:
May be `const`

Reply:
Also just wanted to say in case you are not aware, you can rename variables across the file intelligently using the refactor option. I think F2 is the button shortcut for VS Code, but please try to understand how it works and use it always rather than text comparison and replacement!

Reply:
@tr0mbl3y like mentioned by kenny don't use var. It is being deprecated instead use let & const for the newer ES5+ features.